README.md

```diff
@@ -18,7 +18,7 @@ Enter the value returned by “matlabroot” in the field named “MATLAB root
   ![new_enter_matlabroot](https://user-images.githubusercontent.com/47204011/55624374-45097e00-57c3-11e9-96e1-5fa0fc966767.png)
   
 #### Configuring “Automatic” Option
-This option finds tests written using the MATLAB unit testing framework and/or Simulink Test and runs them. If the code is organized using projects, it will locate all test files in the project that have been classified as "Test". If the code does not leverage projects or uses a MATLAB version prior to R2019a, the plugin will discover all tests in the current Jenkins workspace including subfolders. 
+This option finds tests written using the MATLAB unit testing framework and/or Simulink Test and runs them. If the code is organized using projects, it will locate all test files in the project that have been classified as "Test". If the code does not leverage projects or uses a MATLAB version prior to R2019a, the plugin will discover all tests in the current Jenkins workspace including subfolders. When using "Automatic" option, It is mandatory for user to make sure that, appropriate SCM configuration is done before build step execution, which pulls the latest code from SCM into the Jenkins workspace. If SCM is not used to maintain the code then, user should make sure to add appropriate script, as part of build step before MATLAB plugin to copy the code in Jenkins workspace.
```

Comment:
For me "MATLAB unit testing framework" make sense as we could still have legacy functional test which does not use MATLAB unit testing framework. and such tests also should follow the mandatory naming conventions of adding "test" either at the start or end of the test file name. 

I have updated the para as per suggestion with little changes . 

I agree to use "you" and "your", suggest otherwise 

This documentation has been reviewed internally by all team members and by @mw-hrastega once before last commit.